dynawo/sources/Models/Modelica/Dynawo/Examples/Nordic/Grid/BaseClasses/NetworkWithAlphaBetaLoads.mo

```diff
@@ -135,296 +135,335 @@ model NetworkWithAlphaBetaLoads "Nordic test grid with buses, lines, shunts and
-  final parameter Types.Angle UPhase0_Load_62 = SIunits.Conversions.from_deg(-57.18);
-  final parameter Types.VoltageModule U0Pu_Load_62 = 1.0002;
-  final parameter Types.ComplexPerUnit u0Pu_Load_62 = ComplexMath.fromPolar(U0Pu_Load_62, UPhase0_Load_62);
-  final parameter Types.ComplexPerUnit i0Pu_Load_62 = ComplexMath.conj(s0Pu_Load_62 / u0Pu_Load_62);
-  // Load_63 init values:
-  //s0Pu, i0Pu in SnRef, receptor convention
-  final parameter Types.PerUnit P0Pu_Load_63 = 590.0 / Electrical.SystemBase.SnRef;
-  final parameter Types.PerUnit Q0Pu_Load_63 = 264.6 / Electrical.SystemBase.SnRef;
-  final parameter Types.ComplexPerUnit s0Pu_Load_63 = Complex(P0Pu_Load_63, Q0Pu_Load_63);
-  final parameter Types.Angle UPhase0_Load_63 = SIunits.Conversions.from_deg(-53.49);
-  final parameter Types.VoltageModule U0Pu_Load_63 = 0.9992;
-  final parameter Types.ComplexPerUnit u0Pu_Load_63 = ComplexMath.fromPolar(U0Pu_Load_63, UPhase0_Load_63);
-  final parameter Types.ComplexPerUnit i0Pu_Load_63 = ComplexMath.conj(s0Pu_Load_63 / u0Pu_Load_63);
-  // Load_71 init values:
-  //s0Pu, i0Pu in SnRef, receptor convention
-  final parameter Types.PerUnit P0Pu_Load_71 = 300.0 / Electrical.SystemBase.SnRef;
-  final parameter Types.PerUnit Q0Pu_Load_71 = 83.8 / Electrical.SystemBase.SnRef;
-  final parameter Types.ComplexPerUnit s0Pu_Load_71 = Complex(P0Pu_Load_71, Q0Pu_Load_71);
-  final parameter Types.Angle UPhase0_Load_71 = SIunits.Conversions.from_deg(-7.8);
-  final parameter Types.VoltageModule U0Pu_Load_71 = 1.0028;
-  final parameter Types.ComplexPerUnit u0Pu_Load_71 = ComplexMath.fromPolar(U0Pu_Load_71, UPhase0_Load_71);
-  final parameter Types.ComplexPerUnit i0Pu_Load_71 = ComplexMath.conj(s0Pu_Load_71 / u0Pu_Load_71);
-  // Load_72 init values:
-  //s0Pu, i0Pu in SnRef, receptor convention
-  final parameter Types.PerUnit P0Pu_Load_72 = 2000.0 / Electrical.SystemBase.SnRef;
-  final parameter Types.PerUnit Q0Pu_Load_72 = 396.1 / Electrical.SystemBase.SnRef;
-  final parameter Types.ComplexPerUnit s0Pu_Load_72 = Complex(P0Pu_Load_72, Q0Pu_Load_72);
-  final parameter Types.Angle UPhase0_Load_72 = SIunits.Conversions.from_deg(-6.83);
-  final parameter Types.VoltageModule U0Pu_Load_72 = 0.9974;
-  final parameter Types.ComplexPerUnit u0Pu_Load_72 = ComplexMath.fromPolar(U0Pu_Load_72, UPhase0_Load_72);
-  final parameter Types.ComplexPerUnit i0Pu_Load_72 = ComplexMath.conj(s0Pu_Load_72 / u0Pu_Load_72);
-  final parameter SIunits.Impedance XBase_130 = 130 ^ 2 / Electrical.SystemBase.SnRef;
-  final parameter SIunits.Impedance XBase_220 = 220 ^ 2 / Electrical.SystemBase.SnRef;
-  final parameter SIunits.Impedance XBase_400 = 400 ^ 2 / Electrical.SystemBase.SnRef;
+  // load_01 init values:
+  // s0Pu, i0Pu in SnRef, receptor convention
+  final parameter Types.ActivePowerPu P0Pu_load_01 = 6;
+  final parameter Types.ReactivePowerPu Q0Pu_load_01 = 1.482;
+  final parameter Types.VoltageModulePu U0Pu_load_01 = 0.9988;
```

Comment:
You need to make sure that this values (U0Pu) are updated for case B as they determine the behaviour of the alpha-beta loads ! So maybe a final parameter is not ideal here ?